# automatically add key frames if turned on
uint8 addKeyFrames
---

